### ISSUE TITLE
Improve design of /new project start-page for onboarding flow

### ISSUE THREAD

**Issue by jldec** [gitpod-io/gitpod#6875]
The start page of the New Project onboarding flow contains some rough edges.

## When app is not installed (GitHub only)
(see screenshot below)

- [x] Replace _Select a Git repository..._ with an explanation of what a Project is (under "New Project")  
  > Projects help you manage prebuilds and  
  > workspaces for your repository. [Learn more](https://www.gitpod.io/docs/teams-and-projects)
- [x] Show a label above the dropdown:  
  > User or organization on github.com. (change)  
- [x] Show user account in the dropdown.
- [x] Replace the box with "No Access..." with a friendlier explanation:    
  > Additional authorization is
  > required for our GitHub App to    
  > watch your repositories and 
  > trigger prebuilds.
  `Configure Gitpod App`
- [x] Remove _Repository not found?_ line which is not relevant before the App has been installed.
- [x] Remove _Missing organizations?_ line which is too difficult to understand.
  (add to dropdown in separate issue) 
- [x] Remove _or open a New Workspace with an example repository._ note at the very bottom.

![Screenshot 2021-12-10 at 11 46 11](https://user-images.githubusercontent.com/849592/145569511-8faaed8f-f641-4e7a-9e1a-10653e5866ec.png)

## When repositories can be listed
(see screenshot below)

- [x] Show the same explanation of what a Project is (under "New Project")  
  > Projects help you manage prebuilds and  
  > workspaces for your repository. [Learn more](https://www.gitpod.io/docs/teams-and-projects)
- [x] Show (existing from before) instruction above the dropdown:  
  > Select a Git repository on xxx. (change)  
- [ ] In the list, when a repo is "already taken", provide a hint for how to track down.  
  See #5119. 

## Additional

- [x] Remove _or open a New Workspace with an example repository._ at the bottom. That was a just workaround for bypassing the option to choose from example repos on the website.

**Comment by jldec** [comment]
/schedule

Scheduling this now because it provides aditional context for #5583 
Considering the scenario where a new repo is being created from a template, and the GitHub app is not yet installed, the first section above may be helpful, since it does not suggest selecting any repository from a list.
cc: @jankeromnes

**Comment by roboquat** [comment]
@jldec: Issue scheduled in the meta team (WIP: 0)

<details>

In response to [this](https://github.com/gitpod-io/gitpod/issues/6875#issuecomment-978271525):

>/schedule
>
>Scheduling this now because it provides aditional context for #5583 
>Considering the scenario where a new repo is being created from a template, and the GitHub app is not yet installed, the first section above may be helpful, since it does not suggest selecting any repository from a list.
>cc: @jankeromnes 


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by laushinka** [comment]
Will pick this up. @gtsiolis we can align on the visual design?

**Comment by laushinka** [comment]
@gtsiolis @jldec I'm assuming the tasks for when repositories can be listed depend on #5128? And do we still have pending discussions about what should happen here?

**Comment by laushinka** [comment]
Putting this comment in here for when we talk about the follow-up tasks to this issue https://github.com/gitpod-io/gitpod/pull/7163#discussion_r769484114

**Comment by jldec** [comment]
@laushinka I think we can close this issue - the changes from #7163 cover almost all of the points.
We can work through #5119 separately.

**Comment by laushinka** [comment]
> @laushinka I think we can close this issue - the changes from #7163 cover almost all of the points. We can work through #5119 separately.

@jldec Oh you're right, I didn't realize. Awesome!

**Comment by jldec** [comment]
@gtsiolis feel free to open another issue to improve the UX/UI and design consistency of the repo list.
